src/server/editorServices.ts

```diff
@@ -2209,8 +2214,8 @@ namespace ts.server {
                         // If the file matches, collect its types packages and exclude rules
                         if (rule.types) {
                             for (const type of rule.types) {
-                                if (types.indexOf(type) < 0) {
-                                    types.push(type);
+                                if (typeAcqInclude.indexOf(type) < 0) {

```

Comment:
>if (typeAcqInclude.indexOf(type) < 0) { [](start = 32, length = 39)

We don't otherwise dedup the include or exclude lists, do we?

Reply:
Not if they're user-specified